src/Faker/Provider/sq_AL/Address.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Faker\Provider\sq_AL;
+
+class Address extends \Faker\Provider\Address
+{
+    /**
+     * @see https://en.wikipedia.org/wiki/List_of_cities_and_towns_in_Albania
+     */
+    protected static $city = [
+        'Belsh','Berat','Bulqizë','Cërrik','Delvinë','Devoll','Dibër','Dimal','Divjakë','Dropull','Durrës','Elbasan','Fier','Finiq','Fushë-Arrëz',
+        'Gjirokastër','Gramsh','Has','Himarë','Kamëz','Kavajë','Këlcyrë','Klos','Kolonjë','Konispol','Korçë','Krujë','Kuçovë','Kukës','Kurbin',
+        'Lezhë','Libohovë','Librazhd','Lushnjë','Malësi e Madhe','Maliq','Mallakastër','Mat','Memaliaj','Mirditë','Patos','Peqin','Përmet',
+        'Pogradec','Poliçan','Prrenjas','Pukë','Pustec','Roskovec','Rrogozhinë','Sarandë','Selenicë','Shijak','Shkodër','Skrapa','Tepelenë',
+        'Tiranë','Tropojë','Vau i Dejës','Vlorë','Vorë'
+    ];
+    /**
+     * @see https://en.wikipedia.org/wiki/Category:Streets_in_Tirana
+     */
+
+    protected static $street = [
+        'Tish Daija', 'Avni Rustemi', 'Jordan Misja', 'Agim Nasufi','Naim Frashri', 'Migjeni', 'Pjeter Pograni',
+        'Agim Jashari', 'Isa Boletini','Pjeter Budi', 'Fran Vasa', 'Bajram Curri', 'Azem Galica', 'Albanopoli',
+        'Egnaita', 'Lek Dukagjini', 'Lefter Talo', 'Liri Gero', 'Lidhja e Prizrenit', 'Skernderbej', 'Kupa',
+        'Haki Taha', 'Faik Konica', 'Isuf Xhelili', 'Ali Asllani', 'Fato Berberi', 'Dardania', 'Princ Vidi',
+    ];
+    /**
+     * @see https://en.wikipedia.org/wiki/Foreign_relations_of_Albania
```

Comment:
This is a list of English country names, from English wikipedia. Not localized into Albanian.